src/common/transformations/src/transformations/utils/utils.cpp

```diff
@@ -155,6 +155,35 @@ bool check_for_broadcast(const ov::PartialShape& ref_shape, const ov::PartialSha
     return false;
 }
 
+bool broadcasted_only_channel(const ov::PartialShape& weights_shape, const ov::PartialShape& const_shape) {
+    if (const_shape.size() > weights_shape.size()) {
+        return false;
+    }
+
+    auto weights_shape_it = weights_shape.rbegin();
+    auto const_shape_it = const_shape.rbegin();
+    size_t C_dim = const_shape.size() - 1 - 2;
```

Comment:
what if input is 3d or 5d?
it looks like this calculation will break `C_dim = const_shape.size() - 1 - 2;`